patches/installer/0008-Update-blob-versions-to-use-ref-pkg-versions-to-trac.patch

```diff
@@ -0,0 +1,28 @@
+From 992512755a9df08ccfd531f7507c298aa5b22112 Mon Sep 17 00:00:00 2001
+From: dseefeld <dseefeld@microsoft.com>
+Date: Thu, 29 Oct 2020 16:01:59 +0000
+Subject: [PATCH] Update blob versions to use ref pkg versions to track stable
+ versions
+
+---
+ src/redist/targets/GenerateLayout.targets | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/src/redist/targets/GenerateLayout.targets b/src/redist/targets/GenerateLayout.targets
+index 9e7249d..f1b11d4 100644
+--- a/src/redist/targets/GenerateLayout.targets
++++ b/src/redist/targets/GenerateLayout.targets
+@@ -11,8 +11,8 @@
+ 
+   <PropertyGroup>
+     <!-- Blob storage directories are not stabilized, so these must refer to a package that does not stabilize -->
+-    <AspNetCoreBlobVersion>$(VSRedistCommonAspNetCoreSharedFrameworkx6450PackageVersion)</AspNetCoreBlobVersion>
+-    <CoreSetupBlobVersion>$(MicrosoftNETCoreAppInternalPackageVersion)</CoreSetupBlobVersion>
++    <AspNetCoreBlobVersion>$(MicrosoftAspNetCoreAppRefPackageVersion)</AspNetCoreBlobVersion>
++    <CoreSetupBlobVersion>$(MicrosoftNETCoreAppRefPackageVersion)</CoreSetupBlobVersion>
```

Comment:
This will break in servicing, the ref package doesn't update with every release but the runtime version will. I believe the infra we have putting them in a stable location is what's wrong here (or rather, not matching what the modern infra does anymore). The blob storage virtual folder should always be unstable and only the filename version gets stable.

Reply:
Filed issue https://github.com/dotnet/source-build/issues/1853